services/QuillLMS/engines/evidence/app/models/evidence/activity.rb

```diff
@@ -135,7 +136,7 @@ def flag=(flag)
     end
 
     def invalid_highlights
-      invalid_feedback_highlights
+      invalid_feedback_highlights.concat(invalid_plagiarism_texts)
     end
```

Comment:
I don't think that this matters that much, but it feels like maybe there should be a new name for the combined thing that's highlights plus plagiarism.  Maybe `invalid_related_texts` or something?